google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -346,6 +346,7 @@ gcs.panel.bucket.listing.no.project.selected=To view your buckets select a Cloud
 gcs.panel.bucket.listing.error.loading.buckets=Could not load buckets for selected project.
 gcs.panel.bucket.listing.no.buckets.found=No buckets found for selected project.
 gcs.panel.bucket.listing.loading.text=loading ...
+gcs.panel.bucket.listing.not.logged.in=To view your buckets login to a Cloud Project.
```

Comment:
Technically you're logging into your Google account to view your Cloud projects. Maybe reword this?

Reply:
ah good point. how about: "... login to your Google Cloud Platform account" ?

Reply:
ill check how we word this elsewhere too

Reply:
SGTM, but maybe split "log in" since "login" is a noun for the act of logging in. :)